web/src/emoji_picker.js

```diff
@@ -620,9 +620,6 @@ function register_popover_events($popover) {
 }
 
 export function build_emoji_popover($elt, id) {
-    const template_args = {
-        class: "emoji-info-popover",
-    };
```

Comment:
For this sort of removal commit, a proper commit message should explain why it's correct -- e.g. in this case a sentence "Only one value was ever used."

I know it seems obvious, but if you don't say that, a careful reviewer needs to check this themselves.

Reply:
Got it! Is there anything I should do now, or just to think about this in the future?

Reply:
Yes, next time just extend the commit message when you're touching the PR to answer the question.

(It's very helpful to have this sort of information in the commit message, and not just lost somewhere in the PR review thread, because then it's handy if months from now we discover the change breaks something)